storage/util.go

```diff
@@ -15,10 +15,12 @@ import (
 	"time"
 )
 
-func (c Client) computeHmac256(message string) string {
+func (c Client) computeHmac256(message string) (string, error) {
 	h := hmac.New(sha256.New, c.accountKey)
-	h.Write([]byte(message))
-	return base64.StdEncoding.EncodeToString(h.Sum(nil))
+	if _, err := h.Write([]byte(message)); err != nil {
```

Comment:
`errcheck` doesn't like it. In the other hand, `gas` [doesn't really care](https://github.com/GoASTScanner/gas/blob/master/rules/errors.go#L77). Looking at all this again, I actually think `gas` is more reasonable. I think it would be better if instead of `gas -skip=*/arm/*/models.go -skip=*/management/examples/*.go -skip=*vendor* -skip=*/Gododir/* ./...`, I rewrote that line so for all this skipped files I just exclude whatever [rule](https://github.com/GoASTScanner/gas#available-rules) they break. For example, I'm skipping all the `models.go` files because they "break" rule G101.